packages/wrangler/src/traverse-module-graph.ts

```diff
@@ -49,5 +49,6 @@ export default async function traverseModuleGraph(
 		bundleType,
 		stop: undefined,
 		sourceMapPath: undefined,
+		sourceMapMetadata: undefined,
```

Comment:
Ah, I see.

Reply:
Yeah, this is for cases where there _isn't_ a sourcemap (`--no-bundle`, essentially)